resolver/serviceResolver.go

```diff
@@ -428,16 +428,20 @@ func (resolver *serviceResolver) verifyCodesReturningGuardian(
 		return core.GuardianInfo{}, nil, err
 	}
 
-	addressBytes := userAddress.AddressBytes()
+	addressBytes, err := resolver.pubKeyConverter.Decode(userAddress)
+	if err != nil {
+		return core.GuardianInfo{}, nil, err
+	}
```

Comment:
not needed, if the above suggestion would be applied

Reply:
I still need this for the `userInfo` though, right?

Reply:
reverted